dallinger/default_configs/global_config_defaults.txt

```diff
@@ -45,4 +45,6 @@ num_dynos_worker = 1
 prolific_api_token = Set your Prolific API token in ~/.dallingerconfig!
 prolific_api_version = v1
 prolific_estimated_completion_minutes = 0
+prolific_project = Project
 prolific_recruitment_config = {}
+prolific_workspace = My Workspace
```

Comment:
Related to my comment below, I think it might be better not to provide a default value for `prolific_workspace`.

Reply:
@pmcharrison This changes the behavior so that the experiment must provide a workspace value. Just want to verify this is a good thing?

Reply:
We have a test which creates and deletes a study. This test will now need a study we know to exist if we want to keep the test.

Reply:
Good point about changing behaviour.... I suppose that ideally if the variable is not provided then the deployment defaults to the primary workspace, like it did before?

Reply:
@pmcharrison I could change it to do that, yes.   Do I need to advertise this change somewhere so others can weigh in with their +1 / -1 to the change?

Reply:
We can double check by asking @jessesnyder and @jacobyn whether they're happy? To recap for them, the proposal is that, to preserve previous behaviour, if the user does not supply a specific Prolific workspace to deploy to (via config) then we will default to the primary workspace.

Reply:
I say we do this for now (restore status quo), and optionally create a separate ticket to force explicitly setting a workspace (and maybe even forcing it to be non-default?), but checking that this has been done locally as part of the other experiment/config sanity checks (`dallinger.command_line.utils.verify_package()`).

Reply:
OK... I'll make the change and want for @jacobyn to comment before pushing it up.

Reply:
@jacobyn approved this change (see main thread) so I'm resolving this conversation.